orca/process.py

```diff
@@ -1,13 +1,22 @@
 from .finder import find_filepath
 from .request_opendap import build_url, request_opendap
+from .utils import setup_logging
 
 
-def process_request(connection_string, unique_id, thredds_base, variable, lat, lon):
+def process_request(
+    connection_string, unique_id, thredds_base, variable, lat, lon, log_level="INFO"
+):
     """Uses orca modules to process output"""
+    logger = setup_logging(log_level)
+
+    logger.info(f"Getting the filepath")
     filepath = find_filepath(connection_string, unique_id)
-    print(f"Got the filepath:{filepath}")
+    logger.info(f"filepath: {filepath}")
```

Comment:
There are a bunch of new logging messages which is good. Something we may want to consider is filtering the information a bit more. There are multiple logging levels that can help us do that. For example messages that are valuable to a user we want to keep in `INFO`, whereas smaller progress comments can be in `DEBUG`. I suggest you go through all the logging messages and consider what kind of information should be provided to the user vs a dev.